lib/order.rb

```diff
@@ -0,0 +1,82 @@
+require_relative 'customer'
+require 'pry'
+require 'csv'
+
+def parse_data(data)
```

Comment:
This should probably be a private helper method for the `Order` class, not outside.